Paintroid/build.gradle

```diff
@@ -15,6 +16,11 @@ android {
         testInstrumentationRunner "pl.polidea.instrumentation.PolideaInstrumentationTestRunner"
 		versionCode 1
 		versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```

Comment:
please test this with jenkins first. I'm not sure if this will work for the automated test. Similar issue with Pocket Code...